examples/experimental/dagster-airlift/dagster_airlift/in_airflow/dagster_operator.py

```diff
@@ -14,97 +15,131 @@
 logger = logging.getLogger(__name__)
 
 
-def compute_fn() -> None:
-    # https://github.com/apache/airflow/discussions/24463
-    os.environ["NO_PROXY"] = "*"
-    dag_id = os.environ["AIRFLOW_CTX_DAG_ID"]
-    task_id = os.environ["AIRFLOW_CTX_TASK_ID"]
-    dagster_url = os.environ["DAGSTER_URL"]
-    return launch_runs_for_task(dag_id, task_id, dagster_url)
-
-
-def launch_runs_for_task(dag_id: str, task_id: str, dagster_url: str) -> None:
-    expected_op_name = f"{dag_id}__{task_id}"
-
-    assets_to_trigger = {}  # key is (repo_location, repo_name, job_name), value is list of asset keys
-    # create graphql client
-    response = requests.post(f"{dagster_url}/graphql", json={"query": ASSET_NODES_QUERY}, timeout=3)
-    for asset_node in response.json()["data"]["assetNodes"]:
-        tags = {tag["key"]: tag["value"] for tag in asset_node["tags"]}
-        # match assets based on conventional dag_id__task_id naming or based on explicit tags
-        if asset_node["opName"] == expected_op_name or (
-            tags.get(DAG_ID_TAG) == dag_id and tags.get(TASK_ID_TAG) == task_id
-        ):
-            repo_location = asset_node["jobs"][0]["repository"]["location"]["name"]
-            repo_name = asset_node["jobs"][0]["repository"]["name"]
-            job_name = asset_node["jobs"][0]["name"]
-            if (repo_location, repo_name, job_name) not in assets_to_trigger:
-                assets_to_trigger[(repo_location, repo_name, job_name)] = []
-            assets_to_trigger[(repo_location, repo_name, job_name)].append(
-                asset_node["assetKey"]["path"]
+class IDagsterOperator(BaseOperator, ABC):
```

Comment:
I think "I" prefix should be preserved for more "pure" abstract base classes that don't have a bunch of implemenation. I also think `BaseDagsterOperator` is more consistent with naming conventions.

While we are at it, I think we should reconsider this name. `DagsterOperator` is a very very general name, and I can imagine building more operators in the future that interact with Dagster in some way. We also have another `DagsterOperator` in the code base, which is confusion.

`ProxyToDagsterOperator` or `MigratedToDagsterOperator` or something?

Reply:
I like ProxyToDagsterOperator.